next permutation in c++

```diff
@@ -0,0 +1,69 @@
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
It's bad practice to import all of the `std` namespace into your program at global scope, because it includes many symbols that have common names and that may conflict with yours. Case in point: your `print` function.
Instead, add the `using` declarations at the top of the function body where you want to use them. Or, if you only use a symbol from `std` once or twice, just fully qualify: `std::cout` instead of `cout`.

For more information on how to use namespaces, see the Google C++ style guide here: https://google.github.io/styleguide/cppguide.html#Namespaces